lambda-buffers-codegen/data/rust-prelude-base.json

```diff
@@ -0,0 +1,73 @@
+{
+  "opaquesConfig": {
+    "Prelude.Map": [
+      "std",
+      "collections",
+      "BTreeMap"
+    ],
+    "Prelude.Set": [
+      "std",
+      "collections",
+      "BTreeSet"
+    ],
+    "Prelude.List": [
+      "std",
+      "vec",
+      "Vec"
+    ],
+    "Prelude.Either": [
+      "std",
+      "result",
+      "Result"
+    ],
+    "Prelude.Maybe": [
+      "std",
+      "option",
+      "Option"
+    ],
+    "Prelude.Bytes": [
+      "std",
+      "vec",
+      "Vec<u8>"
+    ],
+    "Prelude.Text": [
+      "std",
+      "string",
+      "String"
+    ],
+    "Prelude.Char": [
+      "char"
+    ],
```

Comment:
How does this work?

Reply:
I have a type
```hs
data Qualified a
  = Qualified'Builtin a
  | Qualified'LibRef CrateName ModuleName a
  deriving stock (Eq, Ord, Show)
```
When parsing JSON, an array with exactly 1 element is parsed as Builtin, exactly 3 elements as `LibRef`

This was required, as `char` is not imported from any standard library, it is a builtin.